fe/__tests__/components/FailureMask/FailureMask_spec.re

```diff
@@ -0,0 +1,13 @@
+open Jest;
+
+let () =
+  describe(
+    "FailureMask",
+    ExpectJs.(
```

Comment:
To otwiera przestrzeń nazw z expect i toMatchSnapshot ? Tu niewiem czy nie czytelniej by było z openem na górze, bo tutaj ilość zagłębień już itak jest spora